kubernetes/AKS/templates/Azure_file_sc.yaml

```diff
@@ -0,0 +1,14 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: my-azurefile
```

Comment:
NIT: Please use a descriptive name

Reply:
Thanks for your comment. The name of the storageclass has been changed to "persistent-volume-azurefile"